src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/QueryIT.java

```diff
@@ -99,6 +105,20 @@ public void multipleFromTest() throws IOException {
         Assert.assertEquals(14, getTotalHits(response));
     }
 
+    @Test
+    public void selectAllWithFieldThrowsException() throws IOException {
+        String fieldName = "age";
+        exception.expect(ResponseException.class);
+        exception.expectMessage("\"error\": {\n" +
+                "    \"reason\": \"Invalid SQL query\",\n" +
+                "    \"details\": \"Other expressions may not be present in the select list when '*' is used\",\n" +
+                "    \"type\": \"SqlParseException\"");
+        executeQuery(String.format(Locale.ROOT,
+                "SELECT *, %s as t " + "" +
+                        "FROM %s/account ORDER BY %s LIMIT 5",
```

Comment:
Do not use String.format for queries, inline everything. It will improve query readability and allow users to copy-paste it to kibana